src/components/header/header.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './header.css';
+
+export default function Header(props) {
+    return (
+        <div className="header">
+            <h1 className="flip-vertical-right">echo</h1><h1>|</h1><h1 className="slide-in-left">|once</h1>
```

Comment:
you should only have one h1 tag on an html page, wrap this in an h1  with two spans, one flip-vertical-right, one slide-in-left instead of the two h1s